### PR TITLE
fix: theme flashing

### PR DESCRIPTION
**Pull request by harish-sethuraman**
When the site loads and no theme is set in the localstorage the site flashes between dark and light mode (when the system is in dark mode)

### How to reproduce: 
- set system theme as dark mode and clear the site's localstorage
- reload the page the page might flash 

Removed some code that are same as the one we already have inside layout
- https://github.com/eslint/new.eslint.org/blob/824c4bf34390a5781dc69f2ca993a873e37d4e15/src/_includes/layouts/base.html#L67-L74

Before: 

https://user-images.githubusercontent.com/32865581/164293001-dac07d15-1a86-4837-8205-a5257f6e42e7.mov

### CODE REVIEW

**Review by snitin315 — COMMENTED** [review]
Looks good to me.

/cc @SaraSoueidan do you want to take a look at this?

File: src/_includes/layouts/base.html [hunk 1]
```diff
@@ -65,12 +65,10 @@
     <link rel="preconnect" href="https://www.googletagmanager.com/">
 
     <script>
-        (function() {
```
Comment (nzakas) [inline src/_includes/layouts/base.html:68]: Seems like we should keep this to avoid creating a global? (Same variable name as in theme.js)

**Review by nzakas — CHANGES_REQUESTED** [review]

File: src/_includes/layouts/base.html (hunk 1 above)
Comment (harish-sethuraman) [reply src/_includes/layouts/base.html:68]: Agreed

Comment (harish-sethuraman) [reply src/_includes/layouts/base.html]: Removed thanks :)

**Comment by SaraSoueidan** [comment]
Good catch! LGTM!

**Review by nzakas — APPROVED** [review]

**Pull request merged**